src/pages/cards.astro

```diff
@@ -0,0 +1,63 @@
+---
+interface Props {
+
+id:number;
+name: string;
+description: string;
+projectUrl: string;
+programYear:number;
+
+}
+const { id, name, programYear,projectUrl,description } = Astro.props;
+
+
+---
+
+        <div class="card">
+            <div class="cardtitle">
+              <h1>{name}</h1>
+            </div>
+          
+            <div class="cardcontainer">
+            <div class="cardcontent">
+              Fall {programYear}
+             </div>
+            </div>
+
+             <div class="description">
+            {description.substring(0,150)+'...'}
```

Comment:
Add the handling for the case if the length of the description is less than 150. Then the `...` should not be displayed.